SEED-platform/seed
# 3216 column description field

#### Any background context you want to provide?
Resolves #3216 

#### What's this PR do?
Adds a column description field to columns that defaults to column name and can be edited as necessary. It also will appear when hovering over column headings in inventory list and inventory list beta. Didn't add to other pages yet.

#### How should this be manually tested?
Add data and then go to organization/column settings to see column description field. Edit one of the column descriptions and save changes. Then go to inventory list and hover over that column heading and ensure that the popup matches the revised column description.
 
#### What are the relevant tickets?
#3216 

#### Screenshots (if appropriate)
